tests/bugfixes/github/test_issue_283.py

```diff
@@ -0,0 +1,183 @@
+# -*- coding: utf-8 -*-
+
+import system_tests
+import os.path
+
+class TestFirstPoC(metaclass=system_tests.CaseMeta):
+    """
+    Regression test for the bug described in:
+    https://github.com/Exiv2/exiv2/issues/283
+    """
+    url = "https://github.com/Exiv2/exiv2/issues/283"
+
+    filename = os.path.join("$data_path", "pocIssue283.jpg")
+    commands = ["$exiv2 " + filename]
+    stdout = [""]
+    stderr = [
```

Comment:
To be honest most of the standard error is not really meaningful for this test case. I would suggest to override the `compare_stderr` method of this class to first run `system_tests.check_no_ASAN_UBSAN_errors` and then only to compare the last two lines, which are two only ones which we actually want to test.

Reply:
I did not know about the existence of `system_tests.check_no_ASAN_UBSAN_errors`. I will definitely use it.